en/docs/setup/analytics/configure-analytics-for-elk-stack.md

```diff
@@ -18,11 +20,31 @@
         ```yaml
         analytics:
           enabled: true
-          type: "ELK"
-          logLevel: "INFO"
+          publishers:
+          - enabled: true
+            type: "elk"
+            logLevel: "INFO"
         ```
 
-3. Redeploy the helm chart with the changes in `values.yaml`.
+You can also set multiple publishers for analytics as follows.
+    ```yaml
+    analytics:
+      enabled: true
+      publishers:
+        - enabled: true
+          configProperties:
+            auth.api.token: <on-prem-key>
+            auth.api.url: "https://analytics-event-auth.choreo.dev/auth/v1"
+          type: "default"
+        - enabled: true
+          type: "elk"
+        - enabled: true
+          type: "moesif"
+          configProperties:
+            moesifToken: <collector-application-id>
+    ```
+
+1. Redeploy the helm chart with the changes in `values.yaml`.
```

Comment:
Shall we fix the numbering here?

Reply:
Fixed in https://github.com/wso2/docs-apk/pull/583 and https://github.com/wso2/docs-apk/pull/584